core/theme/Theme.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace luya\theme;
+
+use luya\helpers\Json;
+use luya\helpers\StringHelper;
+use Yii;
+use yii\base\BaseObject;
+use yii\base\InvalidConfigException;
+
+/**
+ * Extend the yii2 theme component and build up the `$pathMap` depends on given \luya\theme\ThemeConfig which include parents inheritance.
```

Comment:
In order to link to other classes, luya.io will parsed the paths if written in double brackes: {{luya\theme\ThemeConfig}} then it will be clickable. See here: https://github.com/luyadev/luya/blob/master/docs/guide/luya-guideline.md#documentation--guide